values/connectivity/connectivity-raw.gotmpl

```diff
@@ -115,3 +115,70 @@ resources:
 {{- end }} {{/* range clients */}}
 {{- end }} {{/* range */}}
 
+{{- range $client:=$v.egressPublic }}
```

Comment:
This file implements generic test for connectivity. The values for this can be confgiured in `helmfile.tpl/helmfile-connectivity.yaml`. The test is performed manually.
Indeed the values/tests seems to be better place ;)

Reply:
moved there

Reply:
Thanks for hint 👍